### ISSUE TITLE
Plotly 4.2 breaks figure_factory (when scikit-image isn't installed)

### ISSUE THREAD

**Issue by ethanopp** [plotly/plotly.py#1829]
`scikit-image` needs to be added to dependencies

```
C:\Users\OPPENE\PyCharm\Projects\fitness\venv\Scripts\python.exe C:/Users/OPPENE/PyCharm/Projects/fitness/index.py
Traceback (most recent call last):
  File "C:/Users/OPPENE/PyCharm/Projects/fitness/index.py", line 6, in <module>
    from pages import performance, home, settings, power
  File "C:\Users\OPPENE\PyCharm\Projects\fitness\pages\home.py", line 5, in <module>
    import plotly.figure_factory as ff
  File "C:\Users\OPPENE\PyCharm\Projects\fitness\venv\lib\site-packages\plotly\figure_factory\__init__.py", line 27, in <module>
    from plotly.figure_factory._ternary_contour import create_ternary_contour
  File "C:\Users\OPPENE\PyCharm\Projects\fitness\venv\lib\site-packages\plotly\figure_factory\_ternary_contour.py", line 11, in <module>
    from skimage import measure
ModuleNotFoundError: No module named 'skimage'

Process finished with exit code 1
```

**Comment by nicolaskruchten** [comment]
Thanks for bringing this up, it's a regression and we'll fix it!

**Comment by nicolaskruchten** [comment]
Our intention was not to have *all* figure factories depend on scikit-image, just the ternary contours one I believe.

The workaround for 4.2 until the fix is out is just to install the needed dependencies *if you are using a figure factory*.

**Comment by ethanopp** [comment]
Yep that's what I did :)

**Comment by nicolaskruchten** [comment]
Fixed in 4.2.1
